Game logic : Tic Tac Toe Game

yokoworks opened the pull request:
- Added Player and Board classes in lib
- Organized the game loop by creating some methods
- Removed unnecessary comments
- Added attr_reader and attr_accessor to Classes' variables
- Added the require_relative to main.rb in order to call Classes' files

DeliceLydia committed:
Add board, player classes

Commit:
Add methods and loop on main.rb

PhillipUg reviewed (approved):
# Status: Project Approved! 🥇 

Congratulations on implementing all the requirements for this milestone. :+1:

- [OPTIONAL]: Consider moving some of the logic from the `main.rb` into the `lib` folder. For instance, `validate_position` can be moved to `board.rb`. switching and getting current player can be done in `player.rb`

You can merge this branch into the development branch when you're ready and go on to your next milestone.
:bulb:  Do not forget to submit your project completion form after merging this branch.

All the best team,

For any assistance, you can contact me on slack as **Phillip Musiime**

Commit:
Update some methods, game title and description

- Add display_instruction method to give players description
- Update the display_title method to attract players more
- Move switched_player method from main.rb to player.rb
